src/main/java/seedu/duke/Parser.java

```diff
@@ -100,15 +100,15 @@ public static Command checkAction(String inputCommand) throws NumberFormatExcept
                         description += inputs[i] + " ";
                     }
                 }
+
                 commandType = new ProjectCommand(description);
-                break;
             } else {
-                System.out.println("Not in Project View!"); //----------REPLACE WITH EXCEPTION
+                throw new DukeExceptions("Add Task"); // REPLACED WITH EXCEPTION
```

Comment:
I feel that this part is confusing. I understand that it is supposed to throw an error, however the exception message is "add task", maybe change it to "add task error"?

Reply:
Ya Ive left similar comments in your previous pull request, do check it out.

Reply:
Yes even i thought it would confusing because of the way the code is structured, but there's a reason i kept it like this, will explain it to you guys over a zoom session